src/components/edit-and-share/edit-audio/changeImage/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ImageAudio from '../../../ImageAudio';
+import { Upload, message } from 'antd';
+import 'antd/dist/antd.css';
+import './style.css';
+
+function getBase64(img, callback) {
+  const reader = new FileReader();
+  reader.addEventListener('load', () => callback(reader.result));
+  reader.readAsDataURL(img);
+};
+
+function beforeUpload(file) {
+  const isJpgOrPng = file.type === 'image/jpeg' || file.type === 'image/png';
+  if (!isJpgOrPng) { message.error('You can only upload JPG/PNG file!'); };
+  const isLt2M = file.size / 1024 / 1024 < 2;
+  if (!isLt2M) { message.error('Image must smaller than 2MB!'); };
+  return isJpgOrPng && isLt2M;
+};
+
+class ChangeImage extends React.Component {
+  state = {
+    loading: false,
+  };
+
+  handleChange = info => {
+    if (info.file.status === 'uploading') {
+      this.setState({ loading: true });
+      return;
+    }
+    if (info.file.status === 'done') { 
+      getBase64(info.file.originFileObj, imageUrl =>
+        this.setState({ imageUrl, loading: false }));
+    }
+  };
+  render() {
+    const uploadButton = (<div>{this.state.loading ? <ImageAudio /> : <ImageAudio />}</div>
+    );
```

Comment:
Por como están escritos todos los demás archivos, te recomendaría que ese componente lo muevas al JSX del return que tienes abajo. También, ten presente que estás haciendo una validación innecesaria:
```suggestion
    const uploadButton = (<div><ImageAudio /></div>);
```
Fíjate que estás validando si `this.state.loading` y en ambos casos (cuando es `true` o `false`) estás devolviendo el mismo componente (`<ImageAudio />`).

Reply:
Listo